tests/virtualenv_tools_test.py

```diff
@@ -61,6 +61,18 @@ def test_already_up_to_date(venv, capsys):
     assert out == 'Already up-to-date: {0} ({0})\n'.format(venv.before)
 
 
+def test_bourne_shell_exec(venv, capsys):
+    venv.before.join('bin').join('bourne.py').write(
+        "#!/bin/sh\n"
+        f"""'''exec' {venv.before.strpath}/bin/python "$0" "$@"\n"""
+        "' '''\n")
+
+    run(venv.before, venv.after)
+    out, _ = capsys.readouterr()
```

Comment:
can you `pre-commit install && pre-commit run --all-files` we're in the middle of getting this set up in CI

Reply:
Yes